code/modules/unit_tests/initialize_sanity.dm

```diff
@@ -0,0 +1,11 @@
+/datum/unit_test/initialize_sanity/Run()
+	if(length(SSatoms.BadInitializeCalls))
+		Fail("Bad Initialize() calls detected. Please read logs.")
```

Comment:
you cannot see the logs in CI runs, this log_world calls below should be to_chat(world) probably

Reply:
Or, it should call Fail repeatedly, once for each type of init failure and the atoms responsible

Reply:
@Cyberboss @TiviPlus please change it to use Fail repeatedly

if that isn't quite suitable, then use log_test(text)

Reply:
log world goes to stderr which you *can* see in CI.

initialize x subsystem in x seconds is a log_world and you can see those.

Reply:
this printed nothing to the CI and log_test does both a send_text and log call